index.js

```diff
@@ -50,7 +59,18 @@ function getAllMovieTitles() {}
  *  checkIfAnyMovieHasRating(movies, "R");
  *  //> false
  */
-function checkIfAnyMovieHasRating() {}
+function checkIfAnyMovieHasRating(movies, rating) {
```

Comment:
```suggestion
function checkIfAnyMovieHasRating(movies, rating = "G") {
```
This is the syntax for setting a default value on a parameter. This way, rating defaults to be equal to "G" if a rating doesn't get passed in as an argument.